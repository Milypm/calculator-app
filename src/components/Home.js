import React from 'react';
import Navbar from './Navbar';
import '../style/style.css';

const Home = () => (
  <div className="first-div">
    <Navbar />
    <div className="home-div">
      <h2 className="home-h2">Welcome to our page!</h2>
      <p>
        This site idea came to us because we wanted to provide a simple, practical
        and efficient solution for doing calculations on the web.
        <br />
        <br />
        We love Math and we know how important it is in our busy current lives to
        have an easy way to execute operations on a friendly environment.
        <br />
        <br />
        Please, go ahead and save us on your Favorites tab!
      </p>
    </div>
  </div>
);

export default Home;
