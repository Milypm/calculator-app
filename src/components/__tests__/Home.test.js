import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Home from '../Home';

describe('Home component', () => {
  test('renders Home component', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
