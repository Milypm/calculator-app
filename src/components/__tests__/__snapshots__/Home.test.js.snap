// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component displays specific text on the screen 1`] = `
<div
  className="first-div"
>
  <div
    className="header-navbar"
  >
    <div
      className="header-icon"
    >
      <h1>
        Math Magicians
      </h1>
      <i
        className="fas fa-hat-wizard"
      />
    </div>
    <ul
      className="nav-ul"
    >
      <li
        className="li-separator"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="li-separator li-calc"
      >
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote-of-the-day"
          onClick={[Function]}
        >
          Quote of the Day
        </a>
      </li>
    </ul>
  </div>
  <div
    className="home-div"
  >
    <h2
      className="home-h2"
    >
      Welcome to our page!
    </h2>
    <p>
      This site idea came to us because we wanted to provide a simple, practical and efficient solution for doing calculations on the web.
      <br />
      <br />
      We love Math and we know how important it is in our busy current lives to have an easy way to execute operations on a friendly environment.
      <br />
      <br />
      Please, go ahead and save us on your Favorites tab!
    </p>
  </div>
</div>
`;

exports[`Home component renders Home component 1`] = `
<div
  className="first-div"
>
  <div
    className="header-navbar"
  >
    <div
      className="header-icon"
    >
      <h1>
        Math Magicians
      </h1>
      <i
        className="fas fa-hat-wizard"
      />
    </div>
    <ul
      className="nav-ul"
    >
      <li
        className="li-separator"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="li-separator li-calc"
      >
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote-of-the-day"
          onClick={[Function]}
        >
          Quote of the Day
        </a>
      </li>
    </ul>
  </div>
  <div
    className="home-div"
  >
    <h2
      className="home-h2"
    >
      Welcome to our page!
    </h2>
    <p>
      This site idea came to us because we wanted to provide a simple, practical and efficient solution for doing calculations on the web.
      <br />
      <br />
      We love Math and we know how important it is in our busy current lives to have an easy way to execute operations on a friendly environment.
      <br />
      <br />
      Please, go ahead and save us on your Favorites tab!
    </p>
  </div>
</div>
`;
